### DISCUSSION TITLE
More trouble with including vips ... in  VB.Net

### DISCUSSION THREAD

**Discussion by VicJ-beep** [libvips/libvips#2143] [Q&A]
My question may be somewhat similar to Patkalie's, but I thought I'd better not hijack the thread. 

My very first attempt at a Vips statement:    

    `Dim img As NetVips.Image =  NetVips.Image.NewFromFile("D:\pictures\example.png")`     

throws a run-time exception:  
```
    System.DllNotFoundException: 'Unable` to load DLL 'libvips-42.dll'. 
    An unhandled exception of type 'System.DllNotFoundException' occurred in NetVips.dll
    Unable to load DLL 'libvips-42.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E).
```
I'm using Visual Studio 2019 and .NET Framework 4.8., with the following packages installed from NuGet:
``` 
   NetVips v.1.2.4
   NetVips.Extensions v.1.0.2
   NetVips.Native.win-x64
```
Do I really need that libvips-42 dll and if so where can I find it? I'd be grateful for advice.

Vic

**Comment by jcupitt** [comment]
Hi @VicJ-beep,

Check the install notes for NetVips.

https://github.com/kleisauke/net-vips#install
